<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="LAPTOP-5C797KKJ" failures="1" tests="2" name="Test" time="10.237" errors="0" timestamp="2022-07-08T09:53:34 IST">
  <testcase classname="TestNG.TestNGClass1" name="verifyTitle" time="0.018"/>
  <testcase classname="TestNG.TestNGClass1" name="verifyUrl" time="0.012">
    <failure type="java.lang.AssertionError" message="expected [https://jquer] but found [https://jqueryui.com/draggable/]">
      <![CDATA[java.lang.AssertionError: expected [https://jquer] but found [https://jqueryui.com/draggable/]
at TestNG.TestNGClass1.verifyUrl(TestNGClass1.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyUrl -->
  <testcase classname="TestNG.TestNGClass1" name="TestCase2" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase2 -->
  <testcase classname="TestNG.TestNGClass1" name="TestCase1" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase1 -->
</testsuite> <!-- Test -->
